examples/subprojects/project_a/dobi.yaml

```diff
@@ -0,0 +1,22 @@
+#
+# project_a dobi file for subprojects example
+# See README.md for docs
+#
+
+#
+# Images
+#
+#
+
+image=project_a_img:
+    image: example/subprojects/project_a
+    context: project_a
```

Comment:
But if you use the context of ., that is actually referring to the parent of project_a and project_b.  That means that Docker is going to slurp the contents of all the projects just to build one.

In my real project, several of these sub-projects are large, and we definitely don't want them bleeding into each other.

Ideally, I'd love for each sub-project's dobi.yaml file to be rooted in its own working directory by default. 

I got into this whole subproject thing because I was wanting to try to do cross-project coordination and building.  The requirements as I see them at the moment are:
* Set up some sane global defaults that can be used by all the projects
* Be able to build one project before another
* Be able to build a project using the output of another

Thoughts?